ios/FluentUI/Navigation/UINavigationItem+Navigation.swift

```diff
@@ -15,6 +30,14 @@ import UIKit
         static var navigationBarShadow: String = "navigationBarShadow"
         static var usesLargeTitle: String = "usesLargeTitle"
         static var customNavigationBarColor: String = "customNavigationBarColor"
+        static var subtitle: String = "subtitle"
+        static var expandsNavigationBarOnTitleAreaTap: String = "expandsNavigationBarOnTitleAreaTap"
+        static var titleChevronStyle: String = "titleChevronStyle"
```

Comment:
is it possible to reuse/converge with @objc(MSFTwoLineTitleViewAccessoryType)  public enum AccessoryType: Int {